### PR TITLE
Add service.loadBalancerIP in order to set static IP

### PR DESCRIPTION
**Pull request by itscaro**

### CODE REVIEW

**Comment by mtyazici** [comment]
Thanks for the PR @itscaro, could you also update the README files for both OS and EE charts?

**Comment by itscaro** [comment]
@mtyazici I've updated README as well, thanks for the reminder

File: stable/hazelcast-enterprise/templates/service.yaml [hunk 1]
```diff
@@ -20,6 +20,9 @@ spec:
   {{- if (and (eq .Values.service.type "ClusterIP") (.Values.service.clusterIP)) }}
   clusterIP: {{ .Values.service.clusterIP }}
   {{- end }}
+  {{- if (and (eq .Values.service.type "LoadBalancer") (.Values.service.loadBalancerIP) }}
```
Comment (mtyazici) [inline stable/hazelcast-enterprise/templates/service.yaml:23]:
```suggestion
  {{- if (and (eq .Values.service.type "LoadBalancer") (.Values.service.loadBalancerIP)) }}
```

File: stable/hazelcast/templates/service.yaml [hunk 2]
```diff
@@ -20,6 +20,9 @@ spec:
   {{- if (and (eq .Values.service.type "ClusterIP") (.Values.service.clusterIP)) }}
   clusterIP: {{ .Values.service.clusterIP }}
   {{- end }}
+  {{- if (and (eq .Values.service.type "LoadBalancer") (.Values.service.loadBalancerIP) }}
```
Comment (mtyazici) [inline stable/hazelcast/templates/service.yaml:23]:
```suggestion
  {{- if (and (eq .Values.service.type "LoadBalancer") (.Values.service.loadBalancerIP)) }}
```

**Review by mtyazici — CHANGES_REQUESTED** [review]
Thanks for the PR

**Comment by itscaro** [comment]
@mtyazici Thanks for your review, good catch :)

**Review by mtyazici — APPROVED** [review]

**Review by SeriyBg — APPROVED** [review]

**Pull request merged**